# Request
IdCubos[] cubes_position
int8[]  cubes_order
---
# Result
int8[] cubes_correct_order
trajectory_msgs/JointTrajectoryPoint[] cubes_trajectories
---
# Feedback
int8 feedback